python/index.py

```diff
@@ -0,0 +1,31 @@
+# meant for Python 3, will not work with Python 2
+import requests # pip install requests
+
+api_key = 'YOUR_API_KEY_HERE' # it should look like this: VF.XXXXXXX.XXXXXX... keep this a secret!
+version_id = 'YOUR_VERSION_ID_HERE' # your voiceflow project versionID, find it under the 'integrations' tab
+
+def interact(user_id, request):
+    # Start a conversation
+    response = requests.post(
+        f'https://general-runtime.voiceflow.com/state/{version_id}/user/{user_id}/interact',
```

Comment:
is name supposed to be user_id?

Reply:
looks fine otherwise

Reply:
yeah I think its better to call it user ID because in practice the implementation should use unique user properties.

It's just called `name` because in this toy example we're just asking the user for their name as the input

Reply:
Are you using user_id for something, or only api key for authentication? It might be a little confusing when trying to figure what to pass in when

Reply:
@DecathectZero Also maybe adding your remark into the comments